api/src/services/mail/index.ts

```diff
@@ -52,7 +53,11 @@ export class MailService {
 
 		const defaultTemplateData = await this.getDefaultTemplateData();
 
-		const from = `${defaultTemplateData.projectName} <${options.from || (env.EMAIL_FROM as string)}>`;
+		let from = `${defaultTemplateData.projectName} <${options.from || (env.EMAIL_FROM as string)}>`;
```

Comment:
do you think this simpler code would work?
```js
let from = `${options.from.name || defaultTemplateData.projectName} <${options.from.address || options.from || (env.EMAIL_FROM as string)}>`;
```

Reply:
Yep also fine, there might be some typescript warnings though

Reply:
oh, additional check would be needed, when `options.from` is undefined, it would fail